webapp/webpack.config.js

```diff
@@ -40,6 +41,7 @@ const config = {
   },
   devServer: {
     contentBase: "./dist",
+    historyApiFallback: true,
```

Comment:
Hva er dette?

Reply:
Slik at routene skal huske dersom man refresher en side for eksempel: local/foods/:id